packages/fxa-content-server/app/scripts/lib/cropper.js

```diff
@@ -44,19 +39,37 @@ function Cropper(options) {
   if (!options.container) {
     throw new Error('A container element is required');
   }
+
   this._setupElements(options.container);
 
+  window.addEventListener(
```

Comment:
This is looking _so much better_, we do need to remove the event listener though, otherwise when you close the cropper and resize settings it's still present. :( 

See my comment here: https://github.com/mozilla/fxa/pull/3763#discussion_r364926864